src/Concordium/Client/Utils.hs

```diff
@@ -130,6 +130,15 @@ credentialIndexFromStringInform s =
            | otherwise -> Left "Credential index must be non-negative."
     Nothing -> Left "Credential index must be an integer between 0 and 255 (inclusive)."
 
+-- |Try to parse a WasmVersion.
+wasmVersionFromStringInform :: String -> Either String Wasm.WasmVersion
+wasmVersionFromStringInform s =
+  case readMaybe s :: Maybe Word32 of
```

Comment:
This is minor, but I think this should be `Maybe Integer` so that some smart alec does not try to input 4294967296 and this succeeds.

Reply:
Also, in cargo-concordium this is supplied as `V0` and V1` so we should probably match these two.